manifests/input.pp

```diff
@@ -56,7 +56,10 @@
 
   $input_template = $filebeat::major_version ? {
     '5'     => 'prospector.yml.erb',
-    default => 'input.yml.erb',
+    default => $input_type ? {
+        'filestream' => 'filestream.yml.erb',
+        default => 'input.yml.erb',
```

Comment:
Does it make more sense to tie the default value to the installed version of Filebeat?

Reply:
Thanks for the feedback, @pcfens.  Sure, I think that makes sense.  I just pushed 8cac808 which uses:

- `input_type`=`filestream` and `filestream.yml.erb` on filebeat 7.16+
- `input_type`=`log` and `input.yml.erb` on filebeat 6 through 7.15
- `input_type`=`log` and `prospector.yml.erb` on filebeat 5 and earlier